src/screens/import-account/import-account.tsx

```diff
@@ -51,8 +54,21 @@ export const ImportAccount = () => {
               </FormCheckbox>
             </View>
             <Label
-              description="I have read and agree to
-the Terms of Usage and Privacy Policy"
+              description={
+                <>
+                  <Text style={styles.labelText}>I have read and agree to</Text>
+                  <View style={styles.labelWrapper}>
+                    <Text style={styles.labelText}>the </Text>
+                    <TouchableOpacity onPress={() => openUrl(termsOfUse)}>
+                      <Text style={styles.link}>Terms of Usage</Text>
+                    </TouchableOpacity>
+                    <Text style={styles.labelText}> and </Text>
+                    <TouchableOpacity onPress={() => openUrl(privacyPolicy)}>
+                      <Text style={styles.link}>Privacy Policy</Text>
+                    </TouchableOpacity>
+                  </View>
+                </>
```

Comment:
You are overwriting all `Label` styles.  
Let's contact Vova, design should be updated or `Label` should not be used in this case